gtk4/README.md

```diff
@@ -16,8 +16,7 @@ Ruby/GTK4 is a Ruby binding of GTK+ 4.
 
 ## Tutorials, examples:
 
-* gtk4/sample/misc: a lot of simple examples.
-* gtk4/sample/gtk-demo: the ruby version of `gtk-demo`.
+* [gtk4/sample](/gtk4/sample): a lot of simple examples.
```

Comment:
```suggestion
* [sample](sample/): a lot of simple examples.
```

Reply:
I can remove the gtk4/ prefix in the text, but I recommend keeping the absolute path in the link so that it can be copied/moved around the repo without breakage (from my repo documentation experience).